picard/cluster.py

```diff
@@ -269,6 +279,30 @@ def cluster(files, threshold):
 
             yield album_name, artist_name, (files[i] for i in album)
 
+    def update_metadata_images(self):
```

Comment:
This is needed since when removing a file from a cluster, we have to iterate over all files and check if we can remove (or not) the covers from that file from the cluster's metadata.images (checking if other files have the same cover image or not).

Also, this one in Cluster is much simpler than the one in Album, it could be moved together but I think that's a decision I wasn't sure you'd like. Would you prefer to move both Album.update_metadata_images and Cluster.update_metadata_images to a static method in Metadata that would get an obj member (since it has to change album.metadata and album.orig_metadata, but only cluster.metadata) and a list of objects to iterate on ? (so for clusters, it would iterate over its files, but for albums it would iterate over tracks/files/unmatched_files). I'm not sure if it's worth to add even more complexity to have a function that works in the general case as opposed to having similar but simpler code in the two classes.